src/Airy/cairy.jl

```diff
@@ -442,7 +402,63 @@ end
                 invt = @fastmath inv(t)
                 Vec{4, T}((reim(out * invt)..., reim(invt)...))
             end
-            return @fastmath levin_transform(l) * sqrt(xsqr) / T(π)^(3//2) 
+            return @fastmath levin_transform(l) * sqrt(xsqr) / T(π)^(3//2)
+        end
+    )
+end
+
+@generated function airybi_levin(x::Complex{T}, ::Val{N}) where {T <: Union{Float32, Float64}, N}
+    :(
+        begin
+            xsqr = sqrt(x)
+            out = zero(typeof(x))
+            t = GAMMA_ONE_SIXTH(T) * GAMMA_FIVE_SIXTHS(T) / 4
+            a = T(0.25) / (xsqr*x)
+
+            l = @ntuple $N i -> begin
+                out += t
+                t *= -3 * a * (i - 5//6) * (i - 1//6) / i
+                invt = @fastmath inv(t)
+                Vec{4, T}((reim(out * invt)..., reim(invt)...))
+            end
+            out = zero(typeof(x))
+            t = GAMMA_ONE_SIXTH(T) * GAMMA_FIVE_SIXTHS(T) / 4
```

Comment:
why the repetition?

Reply:
Just a product of getting it to be accurate right now. This should be able to be combined.

I'm thinking it will actually be great if we could split these functions into a separate function that can compute a sequence from some arbitrary series with a function handle. Therefore, we could avoid all these generated functions (just have two at the bottom that are repeatedly called). What's happening is we are generating all this code which is similar which is increasing compile times but in reality a lot of the instructions are overlapping it's just the series and `cumsum` that are different. The way it's laid out I think is poor. Hopefully, I'll come up with a better way to do that this week.

Reply:
I think I got it now with recent push. Only way I was able to get it to work right was to use @nexprs then combine with @ntuple later. This reduced the computational time by about 35% with recent pushes. Here is new update....
```julia
@generated function airybi_levin(x::Complex{T}, ::Val{N}) where {T <: Union{Float32, Float64}, N}
    :(
        begin

            xsqr = sqrt(x)
            xsqrx = xsqr * x

            t = GAMMA_ONE_SIXTH(T) * GAMMA_FIVE_SIXTHS(T) / 4
            t2 = 1 / t
            a = inv(4*xsqrx)
            a2 = 4 * xsqrx

            s = zero(typeof(x))
            s2 = zero(typeof(x))
            m = 1
            @nexprs $N i -> begin
                s += t
                s2 += t * m
                t *= -a * (3 * (i - 5//6) * (i - 1//6) / i)
                t2 *= -a2 * (i / (3 * (i - 5//6) * (i - 1//6)))
                l_{i} = Vec{4, T}((reim(s * t2)..., reim(t2)...))
                w_{i} = Vec{4, T}((reim(s2 * t2 * m)..., reim(t2 * m)...))
                m *= -1
            end

            l = @ntuple $N i -> l_{i}
            w = @ntuple $N i -> w_{i}

            e = exp(-2/3 * x * sqrt(x))
            return (e*im*levin_transform(l) + 2*levin_transform(w)/e) / (sqrt(T(π)^3) * sqrt(xsqr))
        end
    )
end
```

Might be a way to continue improve this..... but it's come along way already :)